.gitmodules

```diff
@@ -1,9 +1,27 @@
 [submodule "lib/prometheus"]
 	path = lib/prometheus
-	url = https://github.com/deadtrickster/prometheus.erl.git
+	url = https://github.com/ArweaveTeam/prometheus.erl.git
```

Comment:
`git diff` now shows the change in the subproject commit hash. I suppose we need to commit `lib/prometheus` along with this change.

Reply:
Nevermind, I tried some old commit by mistake initially.